docs/manual/howtos/qos.rst

```diff
@@ -12,10 +12,10 @@
 Quality of Service
 ==================
 
-Quality of Service is the collection of different restrictions and expectations called QoSPolicies on the behaviour of the different components of CycloneDDS.
-Some QoSPolicies only affect a single type of DDS entity, whereas others affect multiple, and QoSPolicies on a DDS entity cannot be modified after creating the entity as they affect matching/discovery of entities.
-The QoS used is linked to a type of DDS entity the QoS is used for, e.g.: a SubscriberQoS is not accepted when creating a DataReader.
-The entity-specific QoSes can be found in the `qos` sub-namespace of the same namespace the entity is defined in, whereas the different QoSPolicies are located in the `dds::core::policy` namespace.
+Quality of Service is the collection of restrictions and expectations (called QoSPolicies) that appy to the different components of CycloneDDS.
+Some QoSPolicies only affect a single type of DDS entity, whereas others affect multiple DDS entities. QoSPolicies on a DDS entity can not be modified after creating the entity as they affect matching/discovery of entities.
```

Comment:
This is an oversight on my part: not all QoSes are not used in matching, just to modify the behaviour of the local entity, we need to specify which are those.

Reply:
Do you have a list of the ones that are used in matching?

Reply:
These (so-called 'RxO' QoS-policies) are listed in the table of chapter 2.2.3 of the OMGDDS specification and are:

- Durability
- Presentation
- Deadline
- Latency_budget
- Ownership
- Liveliness
- Reliability
- Destination_order

Reply:
many thanks - added to the text.